lib/exercise_cases.rb

```diff
@@ -1,2 +1,30 @@
 require 'ostruct'
 require 'json'
+
+class ExerciseCase < OpenStruct
```

Comment:
Not an issue for this PR, but `ExerciseCase` should not be an `OpenStruct` and the canonical data should be kept separate from the methods of this class. 

See: [my imagined version of `ExerciseCase`](https://github.com/Insti/xruby/blob/1e54c30f13cbe584a37a0b3bcd636eb17c8b7f39/lib/exercise_testcase.rb) for an example.